python/pyarrow/tests/parquet/test_metadata.py

```diff
@@ -303,28 +320,30 @@ def test_field_id_metadata():
     pf = pq.ParquetFile(pa.BufferReader(contents))
     schema = pf.schema_arrow
 
-    # Expected Parquet schema for reference
-    #
-    # required group field_id=0 schema {
-    #   optional int32 field_id=1 f0;
-    #   optional group field_id=2 f1 (List) {
-    #     repeated group field_id=3 list {
-    #       optional int32 field_id=4 item;
-    #     }
-    #   }
-    #   optional binary field_id=5 f2;
-    # }
-
     field_name = b'PARQUET:field_id'
```

Comment:
You already have it named `field_id` above.

Reply:
Thanks, fixed to use the existing variable.